charts/tfy-karpenter-config/values.yaml

```diff
@@ -119,6 +121,8 @@ karpenter:
     - alias: al2023@latest
     ## @param karpenter.gpuDefaultNodeTemplate.userData User data script for GPU node template. Set this to "default" to let the chart automatically decide this. Set this to "" to disable this injection.
     userData: "default"
+    ## @param karpenter.gpuDefaultNodeTemplate.enableSoci Enable SOCI for the GPU machines.
```

Comment:
```suggestion
    ## @param karpenter.gpuDefaultNodeTemplate.enableSoci Enable SOCI for the GPU machines. Note this flag is only applicable when `userData` is set to `default`
```

Reply:
To me it is odd to give enableSoci more preference over userData, if I set userData to something custom I have to remember to make enableSoci false

Reply:
We can change the approach later if it doesn't look correct